client/src/components/SignUpPage.vue

```diff
@@ -4,16 +4,17 @@
     <div class="container">
```

Comment:
It's a good practice to let user know if he registered successfully or not / if such user already exists

Reply:
The comment already says it requires better handling